### PR TITLE
Update readme with instructions

### LINKED ISSUE #2: Edit the readme
Add the useful informations to start working with the project

### PR DESCRIPTION
**Pull request by momo2555**
pull request #2

### CODE REVIEW

**Comment by momo2555** [comment]
@KaremED  please review my code

File: README.md [hunk 1]
```diff
@@ -1,6 +1,27 @@
...
+curl https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86_64.exe -o miniconda.exe
+start /wait "" miniconda.exe /S
+del miniconda.exe
+```
+
+## Create a new environment
+```
+conda create -n ai-project python=3.12.0
+conda activate ai-project
+python -m pip install -r src/requirements.txt
```
Comment (KaremED) [inline README.md]: why you didn't used the default installer // pip install -r src/requirements.txt

Comment (momo2555) [reply README.md]: Oh right you can use directly the pip instruction but I prefer using the `-m` option python. Which means use python module executable :)

Comment (KaremED) [reply README.md]: use python3 because we use Conda

Comment (momo2555) [reply README.md]: Done

**Review by KaremED — APPROVED** [review]
ok

**Pull request merged**